### ISSUE TITLE
Implement three measures

### ISSUE THREAD

**Issue by InstantWindy** [kmaninis/OSVOS-PyTorch#24]
Hello! I am a newcomer to learning video segmentation. I think your thesis is very good. Do you implement three measures:region similarity,contour accuracy and temporal instability? Thank you !

**Comment by scaelles** [comment]
Hello,

We use the official code released in python and matlab [here](https://github.com/davisvideochallenge).

In the evaluation of DAVIS 2016 the three metrics that you mention are used for evaluation. In DAVIS 2017, region similarity and contour accuracy are used as there are more occlusions and temporal stability is not reliable in such scenario.

**Comment by InstantWindy** [comment]
Thank you very much! I have some doubts about the train_online.py file.The online training here uses a training set to train the network and then test the network using the test set. Not what you said by using the first frame of the test sequence to learn the appearance of the object. So I am a bit confused. thank you !

**Comment by kmaninis** [comment]
Hi, it is the definition of supervised video object segmentation to be given the label of the first frame and train on that. We separate our training in two steps: 1. training on generic foreground labels and 2. during test time, train on the mask of the given object.
